plugins/chat/spec/system/user_chat_preferences_spec.rb

```diff
@@ -67,6 +67,7 @@
       user_preferences_chat_page.visit
       find("#user_chat_quick_reaction_type_frequent").click
       user_preferences_chat_page.save_changes_and_refresh
+      page.refresh # shouldn't be needed
```

Comment:
Not sure if this is left over.

Reply:
no it's intentional, but maybe I should try to remove it kow and see if it still is flakey, might have been related to something else